Don't touch /var/lock/subsys/lldpad when /etc/init.d/lldpad startup.
For debian does not ship /var/lock/subsys/lldpad and /etc/init.d/lldpad 
don't use it either. 
--- a/lldpad.init
+++ b/lldpad.init
@@ -155,14 +155,14 @@
 		$LLDPAD_BIN -k
 		start_daemon $LLDPAD_BIN -d $OPTIONS
 		rc_status -v
-		[ $? -eq 0 ] && touch /var/lock/subsys/lldpad
+		#[ $? -eq 0 ] && touch /var/lock/subsys/lldpad
 		;;
 	stop)
 		[ "$EUID" = "0" ] || exit 4
 		echo -n $"Shutting down $LLDPAD: "
 		killproc $LLDPAD
 		rc_status -v
-		[ $? -eq 0 ] && rm -f /var/lock/subsys/lldpad
+		#[ $? -eq 0 ] && rm -f /var/lock/subsys/lldpad
 		;;
 	status)
 		echo -n "Checking for service $LLDPAD: "
